components/Footer/Footer.js

```diff
@@ -0,0 +1,25 @@
+import DownloadButton from '../DownloadButton';
+import SocialIcon from '../SocialIcon';
+import styles from './Footer.scss';
+
+const Footer = () => (
```

Comment:
Footer needs Terms of Service and Privacy Policy links. Can go below the copyright notice.

Reply:
@deanpress For the Terms of Service and Privacy Policy, where should those link to? Do we need to essentially copy them into this repo from my.nos.app?